### PR TITLE
change nyVedtak api to match the frontend changes

### PR DESCRIPTION
**Pull request by MengZhuNAV**

### CODE REVIEW

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingslagerService.kt [hunk 1]
```diff
@@ -121,7 +123,7 @@ class BehandlingslagerService(
                 )
             }
 
-            return Ressurs.success(data = behandlingVedtak)
+            return fagsakService.hentRestFagsak(behandlingVedtak.behandling.fagsak.id)
```
Comment (henninghaakonsen) [inline src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingslagerService.kt]: Would it be possible to use fagsakId from the request here?

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/FagsakController.kt [hunk 2]
```diff
@@ -35,20 +35,20 @@ class FagsakController(
...
 
     @PostMapping(path = ["/fagsak/{fagsakId}/nytt-vedtak"])
-    fun nyttVedtak(@PathVariable fagsakId: Long, @RequestBody nyttVedtak: NyttVedtak): ResponseEntity<Ressurs<BehandlingVedtak>> {
+    fun nyttVedtak(@PathVariable fagsakId: Long, @RequestBody nyttVedtak: NyttVedtak): ResponseEntity<Ressurs<RestFagsak>> {
         val saksbehandlerId = oidcUtil.getClaim("preferred_username")
 
         logger.info("{} lager nytt vedtak for fagsak med id {}", saksbehandlerId ?: "Ukjent", fagsakId)
 
-        val behandlingVedtakRessurs: Ressurs<BehandlingVedtak> = Result.runCatching { behandlingslagerService.nyttVedtakForAktivBehandling(fagsakId, nyttVedtak, ansvarligSaksbehandler = saksbehandlerId) }
+        val fagsakRess: Ressurs<RestFagsak> = Result.runCatching { behandlingslagerService.nyttVedtakForAktivBehandling(fagsakId, nyttVedtak, ansvarligSaksbehandler = saksbehandlerId) }
```
Comment (henninghaakonsen) [inline src/main/kotlin/no/nav/familie/ba/sak/behandling/FagsakController.kt]: We can just use the name fagsak I think. The Rest part is just to clearify that this is not the relationmodel.

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingslagerService.kt (hunk 1 above)
Comment (MengZhuNAV) [reply src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingslagerService.kt]: yes, it is easier to read👍

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/FagsakController.kt (hunk 2 above)
Comment (MengZhuNAV) [reply src/main/kotlin/no/nav/familie/ba/sak/behandling/FagsakController.kt]: ok👍

**Pull request merged**